integration-tests/src/test/java/uk/gov/di/authentication/api/VerifyMfaCodeIntegrationTest.java

```diff
@@ -281,9 +281,34 @@ void whenAuthAppCodeRetriesLimitExceededForSignInBlockEmailAndReturn400()
 
         assertThat(response, hasStatus(400));
         assertThat(response, hasJsonBody(ErrorResponse.ERROR_1042));
-        assertEquals(0, redis.getSession(sessionId).getRetryCount());
-        assertThat(userStore.isAccountVerified(EMAIL_ADDRESS), equalTo(true));
-        assertThat(userStore.isAuthAppVerified(EMAIL_ADDRESS), equalTo(true));
+        assertEquals(0, redis.getMfaCodeAttemptsCount(EMAIL_ADDRESS, MFAMethodType.AUTH_APP));
+        assertTrue(redis.isBlockedMfaCodesForEmail(EMAIL_ADDRESS));
+        assertTrue(userStore.isAccountVerified(EMAIL_ADDRESS));
+        assertTrue(userStore.isAuthAppVerified(EMAIL_ADDRESS));
+    }
+
+    @Test
+    void
+            whenAuthCodeRetriesLimitReachedButEmailAddressNotBlockedAllowSmsAttemptAndReturn400WithoutBlockingFurtherRetries()
```

Comment:
The test looks good but could be renamed as there is no auth code retry limit during sign up. It's about ensuring that the auth code retry count doesn't impact the SMS retries.

Reply:
I'd forgotten that, but good point; changing now